corehq/form_processor/models.py

```diff
@@ -1130,6 +1130,20 @@ class CaseTransaction(PartitionedModel, SaveStateMixin, models.Model):
     TYPES_TO_PROCESS = (
         TYPE_FORM,
     )
+    TYPE_ACTIONS_ENUM = (
```

Comment:
I think this should only include the case actions in the order we want for sort. I'm specifically concerned that 'TYPE_FORM' is in there since most actions will be `TYPE_FORM | TYPE_X`

Reply:
Yeah, I should have flagged that as something I wasn't sure about.

The reason I included it was that there seemed to be some that were _just_ `TYPE_FORM` and I wasn't entirely sure how to handle that otherwise. I could handle that as a special case at the end?

Reply:
Oh great point

Reply:
2b835fd

Reply:
I think you can also remove the 'rebuild actions' and I'd change the order slightly to "create, update, index, attachment, ledger, close" (basically create first, close last).

Could also rename this to make it's purpose clear, maybe `FORM_TYPE_ACTIONS_ORDER`?

Reply:
I had included rebuild because it's here in the [original order](https://github.com/dimagi/commcare-hq/blob/2b835fdeee7c0ba9fe9188be11d2de8696a46a13/corehq/ex-submodules/casexml/apps/case/const.py#L16). Similarly, that's why I put close where I did as compared to attachment, ledger, and the rebuilds (interpreting `ledger` as the `commtrack` action type.

I also don't see an update action on case transactions? Am I missing it [here](https://github.com/dimagi/commcare-hq/blob/2b835fdeee7c0ba9fe9188be11d2de8696a46a13/corehq/form_processor/models.py#L1104-L1115)?

Changing the name seems reasonable. Enum means the same thing to me, but it might not be as clear to everyone